contracts/src/IRollup.sol

```diff
@@ -237,4 +242,32 @@ interface IRollup {
      * assertion is not the last confirmed assertion.
      */
     function rejectFirstUnresolvedAssertion(address stakerAddress) external;
+
+    /**
+     * @notice Sets the confirmation period
+     * @param newPeriod New confirmation period
+     *
+     */
+    function setConfirmationPeriod(uint256 newPeriod) external;
+
+    /**
+     * @notice Sets the challenge period
+     * @param newPeriod New challenge period
+     *
+     */
+    function setChallengePeriod(uint256 newPeriod) external;
+
+    /**
+     * @notice Sets the minimum assertion period
+     * @param newPeriod New minimum assertion period
+     *
+     */
+    function setMinimumAssertionPeriod(uint256 newPeriod) external;
+
+    /**
+     * @notice Sets the base stake amount
+     * @param newAmount New base stake amount, this can only be decreased
```

Comment:
@param Should only include what the parameter means.
Shift the *this can only be decreased* line to a separate line under @dev